server/resources/migrations/26_cli_auth.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE instant_cli_auth (
```

Comment:
Might want to call this `instant_cli_auths` to keep with the local custom.

Reply:
Nice!  Went with `instant_cli_logins`